website/components/remote-plugin-docs/utils/fetch-plugin-docs.js

```diff
@@ -0,0 +1,61 @@
+const fetch = require('isomorphic-unfetch')
+const parseSourceZip = require('./parse-source-zip')
+const parseDocsZip = require('./parse-docs-zip')
+
+// Given a repo and tag,
+//
+// return [null, docsMdxFiles] if docs files
+// are successfully fetched and valid,
+// where docsMdxFiles is an array of { filePath, fileString } items.
+//
+// otherwise, return [err, null]
+// where err is an error message describing whether the
+// docs files were missing or invalid, with a path to resolution
+async function fetchDocsFiles({ repo, tag }) {
+  // If there's a docs.zip asset, we'll prefer that
+  const docsZipUrl = `https://github.com/${repo}/releases/download/${tag}/docs.zip`
+  const docsZipResponse = await fetch(docsZipUrl, { method: 'GET' })
+  const hasDocsZip = docsZipResponse.status === 200
+  // Note: early return!
+  if (hasDocsZip) return await parseDocsZip(docsZipResponse)
+  // Else if docs.zip is not present, and we only have the "latest" tag,
+  // then throw an error - we can't resolve the fallback source ZIP
+  // unless we resort to calling the GitHub API, which we do not want to do
+  if (tag === 'latest') {
+    const err = `Failed to fetch. Could not find "docs.zip" at ${docsZipUrl}. To fall back to parsing docs from "source", please provide a specific tag instead of "${tag}".`
```

Comment:
```suggestion
    const err = `Failed to fetch. Could not find "docs.zip" at ${docsZipUrl}. To fall back to parsing docs from "source", please provide a specific version tag instead of "${tag}".`
```